Add shortcode content to OP Q3 new components (#25515)
DataDog/documentation: layouts/shortcodes/observability_pipelines/processors/quota.md

1. Optional: Click **Add Field** if you want to set a quota on a specific service or region field.
1. Enter the field name you want to partition by. See the [Partition example](#partition-example) for more information.
1. Click **Add Field** if you want to add another partition.  
    a. Select the **Ignore when missing** if you want the quota applied only to events that match the partition. See the [Ignore when missing example](#example-for-the-ignore-when-missing-option) for more information.  
    b. Optional: Click **Overrides** if you want to set different quotas for the partitioned field.  
        - Click **Download as CSV** for an example of how to structure the CSV.  
        - Drag and drop your overrides CSV to upload it. You can also click **Browse** to select the file to upload it. See the [Overrides example](#overrides-example) for more information.

#### Examples

##### Partition example

Use **Partition by** if you want to set a quota on a specific service or region. For example, if you want to set a quota for 10 events per day and group the events by the `service` field, enter `service` into the **Partition by** field.

> ##### Example for the ignore when missing option

Reviewer: ```suggestion
##### Example for the **Ignore when missing** option
```
I'm not actually sure whether bold works properly in headers, in which case maybe quotes are acceptable just for readability?

Select **Ignore when missing** if you want the quota applied only to events that match the partition. For example:

If the Worker receives the following set of events:

```
{"service":"a", "source":"foo", "message": "..."}
{"service":"b", "source":"bar", "message": "..."}
{"service":"b", "message": "..."}
{"source":"redis", "message": "..."}
{"message": "..."}
```

> And the **Ignore when missing** is selected, then the Worker separates the logs into two sets:

Reviewer: ```suggestion
And the **Ignore when missing** is selected, then the Worker:
```

> - one for logs with `service:a` and `source:foo`

Reviewer: ```suggestion
- creates a set for logs with `service:a` and `source:foo`
```

> - one for logs with `service:b` and `source:bar`

Reviewer: ```suggestion
- creates a set for logs with `service:b` and `source:bar`
```

> - and ignores the last three events.

Reviewer: ```suggestion
- ignores the last three events
```

Reviewer: Just for parallel structure
